controllers/v1/reportsController.js

```diff
@@ -3012,4 +3017,64 @@ module.exports = class Reports {
       }
     });
   }
+/**
+ * Get submission report.
+ * @method
+ * @name submissionReport
+ * @param {Object} req - requested data.
+ * @param {String} req.query.submissionId - The ID of the submission.
+ * @returns {JSON} JSON response consisting of submission report details.
+ */
+
+/**
+ * Sample Response:
+ * {
+ *   "message": [
+ *     {
+ *       "order": "TSD001_1604483265440-1610349652538",
+ *       "question": "Is electrical wiring / boards closed and insulated?",
+ *       "responseType": "radio",
+ *       "answers": ["R2"],
+ *       "chart": {},
+ *       "instanceQuestions": [],
+ *       "criteriaName": "Survey and Feedback",
+ *       "criteriaId": "5ffbfc5469a1847d4286dfd0",
+ *       "optionsAvailableForUser": [
+ *         { "value": "R1", "label": "Yes" },
+ *         { "value": "R2", "label": "No" }
+ *       ]
+ *     },
+ *     {
+ *       "order": "TSD002_1604483265440-1610349652539",
+ *       "question": "Name of the school?",
+ *       "responseType": "text",
+ *       "answers": ["t"],
+ *       "chart": {},
+ *       "instanceQuestions": [],
+ *       "criteriaName": "Survey and Feedback",
+ *       "criteriaId": "5ffbfc5469a1847d4286dfd0"
+ *     }
+ *   ],
+ *   "status": 200
+ * }
+ */
+
+submissionReport = async function (req) {
+  return new Promise(async function (resolve, reject) {
+    try {
+
+      let generateSubmissionReport = await reportsHelper.surveySubmissionReport(req);
+      resolve(generateSubmissionReport);
+
+    } catch (err) {
+      console.log(err)
```

Comment:
@borkarsaish65 remove console if not required

Reply:
done